packages/libncurses/libncurses.spec

```diff
@@ -1,14 +1,14 @@
-Name: %{_cross_os}ncurses
+Name: %{_cross_os}libncurses
 Version: 6.2
 Release: 1%{?dist}
 Summary: Ncurses libraries
 License: X11
 URL: https://invisible-island.net/ncurses/ncurses.html
 Source0: https://invisible-mirror.net/archives/ncurses/ncurses-%{version}.tar.gz
-Patch1: ncurses-config.patch
-Patch2: ncurses-libs.patch
-Patch3: ncurses-urxvt.patch
-Patch4: ncurses-kbs.patch
+Patch1: libncurses-config.patch
+Patch2: libncurses-libs.patch
+Patch3: libncurses-urxvt.patch
+Patch4: libncurses-kbs.patch
```

Comment:
nit: I'd prefer to keep the patch filenames the same. There are no patch headers but they originate from Fedora's packaging of ncurses.